### DISCUSSION TITLE
Zniffer cost

### DISCUSSION THREAD

**Discussion by mundschenk-at** [zwave-js/zwave-js#4475] [Q&A]
Hi, I might be able to buy a genuine Zniffer for around 100 Euros (someone doesn't need their HW SDK anymore). Is that worthwhile, or am I better off buying a suitable stick and flashing the Zniffer firmware? (As I understand it, a "real" Zniffer does not have frequency filters for the Z-Wave region installed, so it's a bit more universal.)

**Comment by AlCalzone** [comment]
To be honest, I'm not sure. I do know that Silabs' UZB3 (ACC-UZB3-E-STA) works fine as a Zniffer and last I checked that was around 40-50€.
